### PR TITLE
CDOT Configuration and Edges

### PR DESCRIPTION
**Pull request by payneBrandon**
This PR contributes the CDOT configuration of the PPM module back to the main JPO repository. It includes various configuration changes (separated from the main files), documentation updates, as well as the new `*.edges` file currently used for privacy geofencing within Colorado. The updates to `docs/installation.md` describe the new files.

### CODE REVIEW

**Review by TonyEnglish — APPROVED** [review]
thank you, looks good

**Comment by payneBrandon** [comment]
Additional updates to Dockerfile's for locking in a release version of librdkafka: 
- install cmake from apt, rather than wget (significantly speeds up image build with Docker)
- moved away from clone of librdkafka library for a wget of the release zip
   - note this is recommended for production systems by the librdkafka library

File: Dockerfile.cdot [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Add test data. This changes frequently so keep it low in the file.
+ADD ./docker-test /cvdi-stream/docker-test
+
+# change to ppm_data directory
+WORKDIR /ppm_data
+
+# add needed files
+ADD ./data/I-70_I-25_CO-470.edges ./
+ADD ./config/cdot_ppmBsm.properties ./
```
Comment (dan-du-car) [inline Dockerfile.cdot:35]: I have noticed that we can use the https://github.com/usdot-jpo-ode/jpo-cvdp/blob/fa9c0ab7044cfc51bc0b30712bca72a7d5cd9321/Dockerfile-nsv#L25 environment variables from docker-compose.yml https://github.com/usdot-jpo-ode/jpo-ode/blob/8f85728ad5472869b8f70d830f56865d56fdd194/docker-compose.yml#L115

File: Dockerfile.cdot [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+RUN export LD_LIBRARY_PATH=/usr/local/lib && mkdir /cvdi-stream-build && cd /cvdi-stream-build && cmake /cvdi-stream && make
+
+# Add test data. This changes frequently so keep it low in the file.
+ADD ./docker-test /cvdi-stream/docker-test
+
+# change to ppm_data directory
+WORKDIR /ppm_data
+
+# add needed files
+ADD ./data/I-70_I-25_CO-470.edges ./
```
Comment (dan-du-car) [inline Dockerfile.cdot:34]: Maybe use the shared volume in the docker-compose.yml instead of hard coded file names?

**Comment by payneBrandon** [comment]
A generalized solution is in the CDOT system and a new PR will be coming along with that. Closing this one.

**Pull request closed without merging**